### DISCUSSION TITLE
Wireguard module is required, please install it manually if need it

### DISCUSSION THREAD

**Discussion by jonyskids** [bubuntux/nordlynx#72] [General]
I need a bit of clarity here. (I am a noob.)

Do I need to install wireguard in another container? 

If so, do I make the nordvpn container dependent upon it?

**Comment by fship7** [comment]
it needs to be installed on the host as a kernel module. fyi kernels 5.6 and up already have it built in.

**Reply by aurelized** [reply]
sorry, could you explain how you do this, please?

**Reply by fship7** [reply]
The host needs to support wireguard, either by external or internal kernel module. If you are using kernel 5.6+ then it's probably already installed. If you dont have the kernel module then a place to start is https://www.wireguard.com/compilation/
